### PR TITLE
Deploy cmd

### PR DESCRIPTION
**Pull request by davidsteinsland**
### `nais deploy`

Goal: Simplify deploys by providing sane defaults and error handling.

#### Defaults

| Property        | Default           |
| ------------- | ------------- |
| cluster           | preprod-fss   |
| environment | t0                    |
| namespace   | default            |

Fasit username and password can also be specified using environment variables, 
`NAIS_USERNAME` and `NAIS_PASSWORD`.

#### Sample Jenkins pipeline

```groovy
withCredentials([[
  $class: 'UsernamePasswordMultiBinding', 
  credentialsId: 'naisUser', 
  usernameVariable: 'NAIS_USERNAME', 
  passwordVariable: 'NAIS_PASSWORD']]) {
    sh "nais deploy --app myapp --version myver"
}
```

... will deploy to `preprod-fss` with the following JSON:

```json
{
  "application":"myapp",
  "version":"myver",
  "environment":"t0",
  "zone":"fss",
  "username":"<username>",
  "password":"<password>",
  "namespace":"default"
}
```

---

```
$ ./nais deploy --help
Deploys your application

Usage:
  nais deploy [flags]

Flags:
  -a, --app string           name of your app
  -c, --cluster string       the cluster you want to deploy to (default "preprod-fss")
  -e, --environment string   environment you want to use (default "t0")
  -n, --namespace string     the kubernetes namespace (default "default")
  -p, --password string      the password
  -u, --username string      the username
  -v, --version string       version you want to deploy
  -z, --zone string          the zone the app will be in (default "fss")
```

### CODE REVIEW

**Comment by davidsteinsland** [comment]
yeah, was wondering about this. I can change cluster to `preprod-fss`. is environment t0 ok?
And the "zone" in the deployment request: is this an enum of `fss` and `sbs`, or are there more?

**Comment by jhrv** [comment]
For environment/namespace, maybe default should be default?
Yes, correct in addition to the iapp-zone. I think fss should be default for now.

File: cli/cmd/Deploy.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+)
+
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "Deploys your application",
+	Long:  `Deploys your application`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		clusters := map[string]string{
+			"ci":           "nais-ci.devillo.no",
```
Comment (audunstrand) [inline cli/cmd/Deploy.go]: Do we want CI here at all?

Comment (davidsteinsland) [reply cli/cmd/Deploy.go]: I don't know, is the cluster in use?

Comment (audunstrand) [reply cli/cmd/Deploy.go]: its part of the nais cluster verifiation process. But then again, its a good thing to verify this client as well. So I change my mind. Keep CI

**Pull request merged**